src/app/builders/components/Table/BuildersTable.tsx

```diff
@@ -167,6 +143,11 @@ export const BuildersTable = ({ filterOption }: { filterOption: BuilderFilterOpt
       type: 'SET_COLUMNS',
       payload: DEFAULT_HEADERS,
     })
+    // Set default sorting to backer_rewards (descending)
```

Comment:
If the default sorting is the `backer_rewards` why when we cancel it we dont go back to it?

Instead we deselect everything.